diff --git a/common/kernel/module/version.c b/common/kernel/module/version.c
index 53f43ac..56a6c6b 100644
--- a/common/kernel/module/version.c
+++ b/common/kernel/module/version.c
@@ -51,8 +51,8 @@ int check_version(const struct load_info *info,
 	return 1;
 
 bad_version:
-	pr_warn("%s: disagrees about version of symbol %s\n", info->name, symname);
-	return 0;
+	pr_warn("%s: disagrees about version of symbol %s, but ignore...\n", info->name, symname);
+	return 1;
 }
 
 int check_modstruct_version(const struct load_info *info,
-- 
2.55.0
