diff --git a/common/kernel/module/version.c b/common/kernel/module/version.c
index 53f43ac..56a6c6b 100644
--- a/common/kernel/module/version.c
+++ b/common/kernel/module/version.c
@@ -52,7 +52,7 @@ int check_version(const struct load_info *info,
 
 bad_version:
 	pr_warn("%s: disagrees about version of symbol %s\n", info->name, symname);
-	return 0;
+	return 1;
 }
 
 int check_modstruct_version(const struct load_info *info,
